config/crd/experimental/gateway.networking.k8s.io_gateways.yaml

```diff
@@ -720,6 +720,104 @@ spec:
                             type: object
                           maxItems: 64
                           type: array
+                        frontendValidation:
```

Comment:
Can you add an example to `/examples/experimental` that uses this field to ensure that it's all working as intended?